table/block_builder.cc

```diff
@@ -74,6 +90,9 @@ size_t BlockBuilder::EstimateSizeAfterKV(const Slice& key, const Slice& value)
   estimate += VarintLength(key.size()); // varint for key length.
   estimate += VarintLength(value.size()); // varint for value length.
 
+  /* one more <TAG, restart_index_> pair in bucket */;
+  estimate += data_block_hash_index_builder_ ? sizeof(uint16_t) * 2 : 0;
```

Comment:
Shouldn't this be inside the if condition when a new bucket is added?
Also we don't have a TAG anymore, so may be just one sizeof(uint16_t).

Reply:
fixed, Thank! According to the current design, hash table won't increase size as we do not use bucket anymore.